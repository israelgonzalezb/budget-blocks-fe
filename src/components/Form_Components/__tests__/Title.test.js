
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Title from '../Title';


describe('Title test', () => {
  it('Title should match snapshot', () => {
    const component = renderer.create(<Title
       />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
