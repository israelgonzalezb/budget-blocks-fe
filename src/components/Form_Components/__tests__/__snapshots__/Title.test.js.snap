// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title test Title should match snapshot 1`] = `
<div
  className="logo_name"
>
  <img
    alt="logo"
    className="logo-reg"
    src="Logo.svg"
  />
  <h2
    className="MuiTypography-root sign MuiTypography-h2"
  />
</div>
`;
